### ISSUE TITLE
Test: Quick Tree from a screen reader

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#258361]
Refs: https://github.com/microsoft/vscode/issues/252054

- [x] anyOS @lramos15
- [x] anyOS @alexr00

Complexity: 4

[Create Issue](https://github.com/microsoft/vscode/issues/new?template=blank&body=Testing+%23258361%0A%0A&assignees=TylerLeonhardt)

---

@meganrogge had the great idea that we should have a TPI for testing the Quick Tree from a screen reader perspective.

There's a new Tools picker in town!

<img width="490" height="368" alt="Image" src="https://github.com/user-attachments/assets/a61eed17-62f8-4e00-b3e5-e73bb813a2ff" />

Notable additions:
* icons
* toolsets
* collapse/expand
* tri-state checkboxes

Go ahead and play around with it while using a screen reader and tell me if what you hear makes sense! It should behave somewhere in the middle of a quick pick and a tree.

You can launch it either from Agent mode or from a prompt.md file.
